docs/insomnia/get-started/mock-server/allowlist.md

```diff
@@ -3,60 +3,29 @@ layout: article-detail
 title:  Allowlist for Insomnia
 ---
 
-<!-- for docs rework: revise this page like we normally would -->
+For enterprise users utilizing Insomnia, it's crucial to ensure that specific domains are allowlisted in your network. This step is essential to guarantee that all features of Insomnia, including updates, synchronization, documentation, and necessary backend services work without any hindrance in environments with restricted internet access.
```

Comment:
I don't really see the value in this paragraph. I think it should just kind of introduce the table.